pykilosort/learn.py

```diff
@@ -1198,6 +1198,10 @@ def learnAndSolve8b(ctx):
                 *ndrop
             )
 
+            if sanity_plots:
+                assert plot_widgets is not None
```

Comment:
```suggestion
                assert plot_widgets is not None, "if sanity_plots is set then plot_widgets cannot be None"
```

Reply:
☝🏼 will give a better error message.

Reply:
Nice! I didn't know of this nifty trick, thanks!